.ado/variables/msbuild.yml

```diff
@@ -1,7 +1,7 @@
 variables:
-  MSBuildArchitecture: x64
-  MSBuildPreferredToolArchitecture: x64
-  MSBuildPlatformToolset: v141
+  MSBuildArchitecture: x86
+  MSBuildPreferredToolArchitecture: x86
```

Comment:
my point is we should not need to use x86 tools - they are bound to be worse than x64 (addressable virtual memory capped, e.g., which caused MUX to fail to link when running LTCG)